clojure/fira_code/checks.clj

```diff
@@ -4,20 +4,33 @@
    [fira-code.coll :as coll]
    [fira-code.glyphs :as glyphs]))
 
+(def expected-widths #{"0" 0 1200})
+
 (defn width-ok? [w]
-  (#{"0" 0 1200} w))
+  (expected-widths w))
```

Comment:
Giving name to `#{"0" 0 1200}` is good. I would’ve move it inside the function with let, though, to keep everything local